src/components/Forside/Forside.less

```
@@ -0,0 +1,222 @@
+.forside {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 4rem;
+  color: #3E3832;
+
+  .ie-feil {
+    margin-bottom: 2rem;
+  }
+
+  & > div,
+  h1 {
+    margin-bottom: 2rem;
+  }
+  .header {
+    width: 100%;
+    height: 250px;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: repeat(5, 1fr);
+    grid-template-areas:
+      "."
+      "veileder"
+      "veileder"
+      "veileder"
+      "veileder";
+
+    .banner {
+      height: 199px;
+      background-color: @navLillaLighten60;
+      grid-column-start: 1;
+      grid-column-end: 2;
+      grid-row-start: 1;
+      grid-row-end: 3;
+    }
+    .nav-veileder {
+      grid-area: veileder;
+    }
+    .nav-veileder__snakkeboble {
+      grid-area: snakkeboble;
+    }
+
+    .nav-veileder > .nav-veileder__frame {
+      background: @navLillaDarken20;
+
+      svg {
+        width: 100px;
+        height: 100px;
+      }
+    }
+  }
+
+  .forside__innhold {
+    margin: auto;
+    width: @panel-innhold-bredde;
+    @media @tablet {
+      width: 100%;
+      display: flex;
+    }
+
+    @media @mobile {
+      width: 100%;
+      display: flex;
+    }
+
+    .veileder {
+      margin-bottom: 4rem;
+      display: inline-block;
+    }
+
+    .nav-veileder  {
+      &__snakkeboblePil {
+        height: 0;
+      }
+    }
+
+    .seksjon  {
+      .typo-element {
+        font-size: 20px;
+      }
+
+      .alertstripe__ikon {
+        display: none;
+      }
+    }
+
+    .bekreftCheckboksPanel {
+      background-color: #ffe9cc;
+    }
+
+    .alertstripe__tekst {
+      padding-left: 1rem;
+    }
+  
+    .forside__panel {
```

Siden denne ligger under forside kan man skrive om til:

```suggestion
    &__panel {
```